cves/kernel/CVE-2013-2058.yml

```diff
@@ -1,174 +1,51 @@
-
-  Try to still be specific in your description, but remove project-specific
-  stuff. Remove references to versions, specific filenames, and other jargon
-  that outsiders to this project would not understand. Technology like "regular
-  expressions" is fine, and security phrases like "invalid write" are fine to
-  keep too.
-
-  Your target audience is people just like you before you took any course in
-  security
-description:
-bounty_instructions: |
-  If you came across any indications that a bounty was paid out for this
-  vulnerability, fill it out here. Or correct it if the information already here
-  was wrong. Otherwise, leave it blank.
+description: "The host_start function in drivers/usb/chipidea/host.c in the Linux kernel before 3.7.4 
+  does not properly support a certain non-streaming option. This allows local users to cause a denial 
+  of service (system crash) by sending a large amount of network traffic through a USB/Ethernet adapter."
 bounty:
   amt:
   announced:
   url:
 reviews: []
-bugs_instructions: |
-  What bugs are involved in this vulnerability?
-
-  Please list bug IDs to https://bugzilla.kernel.org/
-
-  Bug ID's can appear in several places:
-    * Mentioned in commit messages
-    * Mentioned in mailing list discussions
-    * References from NVD entry
-    * Various other places
-bugs: []
-fixes_instructions: |
-  Please put the commit hash in "commit" below.
-
-  This must be a git commit hash from the systemd source repo, a  40-character
-  hexademical string/
-
-  Place any notes you would like to make in the notes field.
+bugs: [959210]
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: 929473ea05db455ad88cdc081f2adc556b8dc48f
-  note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
-vcc_instructions: |
-  The vulnerability-contributing commits.
-
-  These are found by our tools by traversing the Git Blame history, where we
-  determine which commit(s) introduced the functionality.
-
-  Look up these VCC commits and verify that they are not simple refactorings,
-  and that they are, in fact introducing the vulnerability into the system.
-  Often, introducing the file or function is where the VCC is, but VCCs can be
-  anything.
-
-  Place any notes you would like to make in the notes field.
+  note: "Manually confirmed"
 vccs:
 - commit: eb70e5ab8f95a81283623c03d2c99dfc59fcb319
-  note: Discovered automatically by archeogit.
-upvotes_instructions: |
-  For the first round, ignore this upvotes number.
-
-  For the second round of reviewing, you will be giving a certain amount of
-  upvotes to each vulnerability you see. Your peers will tell you how
-  interesting they think this vulnerability is, and you'll add that to the
-  upvotes score on your branch.
+  note: "File created with vulnerability in this commit"
 upvotes:
```

Comment:
I am going to give this a 4 because I like vulnerabilities that include hardware.